tagged-localbuild.xml

```diff
@@ -8,63 +8,55 @@
-  <project name="hardware-qcom-media-sm8150" path="vendor/qcom/opensource/media/sm8150" remote="sony" revision="7c57b64bbaa26f5308177e5399956734a3300ff3" upstream="aosp/LA.UM.8.1.r1" dest-branch="aosp/LA.UM.8.1.r1" groups="device"/>
-  <project name="hardware-qcom-media-sm8150" path="vendor/qcom/opensource/media/sm8250" remote="sony" revision="82fd2358795f5db13a9c21dc78b5d96304be2844" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device"/>
-  <project name="hardware-qcom-wlan" path="vendor/qcom/opensource/wlan" remote="sony" revision="6351125ec900f421276213a2a0d336cfc6268042" upstream="master" dest-branch="master" groups="device"/>
-  <project name="json-c" path="vendor/oss/json-c" remote="sony" revision="3b0d70d918bc57153721d45645cb953a493ebe19" upstream="master" dest-branch="master" groups="device"/>
-  <project name="kernel-sony-msm-4.19-common" path="kernel/sony/msm-4.19/common-kernel" remote="sony" revision="0ee7cd4893f7074f36a45af2e98c42163ce1a3d6" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device" clone-depth="1"/>
-  <project name="kernel/configs" revision="e4102999005f8f3c8db1fdacfba80a2c581a72f1" upstream="refs/tags/android-11.0.0_r43" dest-branch="refs/tags/android-11.0.0_r43" groups="vts,pdk"/>
-  <project name="macaddrsetup" path="vendor/oss/macaddrsetup" remote="sony" revision="d2871b2041790f6d291e207bec8b0ef34029ef59" upstream="master" dest-branch="master" groups="device"/>
-  <project name="nfc-NXPNFCC_FW" path="vendor/nxp/NXPNFCC_FW" remote="NXP" revision="08f462368aa9cfd721172cc1facc00a6d325a386" upstream="master" dest-branch="master" groups="device"/>
-  <project name="packages_apps_ExtendedSettings" path="packages/apps/ExtendedSettings" remote="sony" revision="0973b752827008710b5b4ff30128affca34d22e3" upstream="master" dest-branch="r-mr1" groups="device"/>
-  <project name="platform/art" path="art" revision="83cd633bec9586c44d214f01734d29de582882a3" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
-  <project name="platform/bionic" path="bionic" revision="bf65a88e2c5d2edc0b1d32e7d78c1be9e012eeb9" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
-  <project name="platform/bootable/recovery" path="bootable/recovery" revision="6202f19002e8fabdafbf298c23516ba296fde1d5" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
-  <project name="platform/build" path="build/make" revision="c84846c1fc04bf6275e840cdd89b810f5f0adfd5" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk">
+  <project name="QcRilAm" path="vendor/oss/qcrilam" remote="sony" revision="ef51ec65f5609e0fe9180ec3d2d054b72c0b8bee" upstream="master" dest-branch="master" groups="device,local::oss"/>
```

Comment:
For every line in this tagged-localbuild.xml that you changed branch or removed altogether, please do the same in localbuilld.xml